src/Desktop/ColorContrastAnalyzer/ColorPair.cs

```diff
@@ -65,6 +65,8 @@ public Boolean AreVisuallySimilarColors()
          */
         public Boolean IsVisiblySimilarTo(ColorPair otherPair)
         {
+            if (otherPair == null) throw new ArgumentNullException(nameof(otherPair));
```

Comment:
Would returning false make sense here (instead of throwing)?

Reply:
Old code would have thrown, so keeping the throwing seems reasonable

---
In reply to: [311285509](https://github.com/microsoft/axe-windows/pull/130#discussion_r311285509) [](ancestors = 311285509)